mlir/lib/Dialect/MIOpen/Generator/Conv2dGenerator.cpp

```diff
@@ -51,7 +51,7 @@ int Conv2dGenerator::getKernelCount(const char *arguments) {
 
   auto operation = argMap["operation"];
   if (operation == "conv2d") {
-    count = 1;
+    count = 4;
```

Comment:
please improve the logic so it counts based on `dilation` and `stride`. check with @asroy on the actual formula.

Reply:
Yes, this was a placeholder. @asroy what is the formula?

Reply:
+ @asleepzzz for awareness. please supply the formula to correctly count the number of kernels after the split.

Reply:
Still haven't seen this but it can be filled in when the sub-kernels are ready.

Reply:
It's a long weekend in Asia so probably @asleepzzz and @ltqin are not back in the office yet.